src/presentation/react-components/BulkDataImport.tsx

```diff
@@ -0,0 +1,79 @@
+import Title from "antd/lib/typography/Title";
+import { useCallback, useState } from "react";
+import { Alert, Button, Space, Statistic } from "antd";
+import Paragraph from "antd/lib/typography/Paragraph";
+import {
+  buildBulkDataImportResultsViewModel,
+  BulkDataImportViewModel,
+} from "../BulkDataImport";
+import { performBulkDataImport } from "../../api-client/bulkDataImport";
+
+const BulkDataImport = ({
+  readingCount,
+  notehubProjectURL,
+}: BulkDataImportViewModel) => {
+  const [isInProgress, setIsInProgress] = useState(false);
+  const [successMessage, setSuccessMessage] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  const performImport = useCallback(async () => {
+    setIsInProgress(true);
+    setErrorMessage("");
+    setSuccessMessage("");
+
+    const resp = await performBulkDataImport();
+    const { message, isGood } = buildBulkDataImportResultsViewModel(resp);
+
+    if (isGood) setSuccessMessage(message);
+    else setErrorMessage(message);
+
+    setIsInProgress(false);
+  }, []);
+
+  return (
+    <>
+      <Title level={2}>Bulk Data Import</Title>
+      <Space direction="vertical" size="large">
+        <Paragraph>
+          <strong>Import reading history from Notehub</strong> into the Sparrow
+          database. Import is useful to get data from periods when routing was
+          not operational. For example, a period when the Notehub route was not
+          configured yet or when there was downtime on this web app or its
+          database. Bulk Data Import is limited to the Event Retention period of
+          your Notehub subscription. Import is safe to run repeatedly (i.e. it
+          is idempotent).
```

Comment:
Might be worth having some wording in here that says the import might take a while.

Reply:
Added a message that pops up to that effect. 
![image](https://user-images.githubusercontent.com/22032748/164114334-b797d357-cf05-4e53-8eba-b4cb82a533e9.png)